### ISSUE TITLE
minikube hostpath mount permissions

### ISSUE THREAD

**Issue by jlj0n3s** [kubernetes/minikube#3599]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG

Please provide the following details:

**Environment**:

```
minikube version: v0.33.1
OS macOS 10.13.6
VM driver: virtualbox
ISO Version: Boot2DockerURL: file://Users/foo/.minikube/cache/iso/minikube-v0.33.1.iso
Kubernetes: v1.10.11
```
**What happened**:
I'm trying to mount a local directory to be used by a container in kubernetes, but getting this error:

    $ kubectl logs mysql-pd

    chown: changing ownership of '/var/lib/mysql/': Input/output error

**What you expected to happen**:

I should be able to maintain a persistent volume for the mysql database that survives the container shutting down

**How to reproduce it** (as minimally and precisely as possible):

I'm starting up minikube with mounted directory:

    minikube start --mount-string /Users/foo/mysql_data:/mysql_data --mount

deployment.yml

    apiVersion: v1
    kind: Pod
    metadata:
      name: mysql-pd
    spec:
      containers:
      - image: mysql:5.7
        name: mysql-container
        env:
        - name: MYSQL_ROOT_PASSWORD
          value: ""
        - name: MYSQL_ALLOW_EMPTY_PASSWORD
          value: "yes"
        ports:
          - containerPort: 3306
        volumeMounts:
        - mountPath: "/var/lib/mysql"
          name: host-mount
      volumes:
      - name: host-mount
        hostPath:
          path: "/mysql_data"

**Comment by tstromberg** [comment]
I suspect this may be a duplicate of #2290, but won't close this until I have more evidence that this is the case.

**Comment by tstromberg** [comment]
I believe this issue was resolved in the v1.1.0 release. Please try upgrading to the latest release of minikube, and if the same issue occurs, please re-open this bug. Thank you opening this bug report, and for your patience!

**Comment by damorosodaragona** [comment]
Same problem. When i try to mount a local directory to be used by a container in kubernetes, i get this error: `Can't change data directory owner`. But not after 90d of inactivity, but when i try to start it for the first time.
